src/util/core.asm

```diff
@@ -280,16 +286,16 @@ switch_s_plus_2a:
         tay
         iny
         pla
-        sta tmp1
+        sta switchTmp1
```

Comment:
this was just to avoid crashing in case I added too much to the frame and caused an unintended crash. I don't think it needs to be there but it seemed like a reasonable failsafe to add. If it's not worth the extra bytes, it can be removed.